src/containers/other/NFTView/assets/more-vertical.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M12.2538 13C12.8061 13 13.2538 12.5523 13.2538 12C13.2538 11.4477 12.8061 11 12.2538 11C11.7015 11 11.2538 11.4477 11.2538 12C11.2538 12.5523 11.7015 13 12.2538 13Z" stroke="#52CBFF" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
@domw30 @ericlowturing  Can we remove the png file and just have the SVG? Please check for other files as well.